instrumented/nightly-tests/build.gradle.kts

```diff
@@ -112,6 +110,11 @@ dependencies {
     implementation(Dependencies.Libraries.Elmyr)
     implementation(Dependencies.Libraries.OkHttp)
 
+    // Ktor (local server)
+    implementation("io.ktor:ktor:1.4.3")
```

Comment:
it is already used in the sample app as well, so maybe let's pull it up to the `Dependencies` class for a centralized version management?